### ISSUE TITLE
process.env variables appear in UI tooltips, but do not appear in the request

### ISSUE THREAD

**Issue by mikestok** [usebruno/bruno#1256]
I am using bruno 1.5.1 installed on a Mac using homebrew.

When I try and access a variable from the `process.env` in a request header it does not seem to be included in the request. The value appears correctly when I hover over the `{{process.env.MY_SECRET}}` in the headers on the left side of the UI.

![Screenshot 2023-12-20 at 11 38 20 AM](https://github.com/usebruno/bruno/assets/64760/def6ac44-fb4a-4d76-bb81-b4c11f5b3d1b)

Sample collection at https://github.com/mikestok/bruno-process-env-secrets.git

**Comment by de-hru** [comment]
For sure, if hover works, the replacement should work in the request too.

Nevertheless, did you try this?

```
vars {
  my-secret: from-environment
  MY_SECRET: {{process.env.MY_SECRET}}
}
```

**Comment by mikestok** [comment]
That does work for me in terms of getting the right thing into the request. Thank you.

To me the hover behaviour still seems a little odd, this is a good workaround.

![Screenshot 2024-01-04 at 9 54 44 AM](https://github.com/usebruno/bruno/assets/64760/3c02b3fb-fb1b-49f1-a213-db8fc9207a12)

**Comment by mj-h** [comment]
The first process variable is not properly replaced. All subsequent ones work.

My .env-file:
![image](https://github.com/usebruno/bruno/assets/20513065/9dbb40d4-a10a-462d-b97c-6551318029eb)

My request-body (yes, it's a body on a GET, it should still work!):
![image](https://github.com/usebruno/bruno/assets/20513065/0b11727d-03bc-400b-a190-5f2d94bd7834)

My post-response script to log the body:
![image](https://github.com/usebruno/bruno/assets/20513065/86f57435-b8ba-4db8-ba80-4ddbb7a10024)

The log of the body. **Notice that line 1 is empty!**:
![image](https://github.com/usebruno/bruno/assets/20513065/b846750e-50de-426d-a9be-6e7deccbbf35)

----

Here the code to reproduce:
The `request.bru` file:
```
meta {
  name: Get 1
  type: http
  seq: 1
}

get {
  url: https://jsonplaceholder.typicode.com/todos/1
  body: text
  auth: none
}

body:text {
  line1: {{process.env.test1}}
  line2: {{process.env.test1}}
}

script:post-response {
  console.log(req.body) 
}
```

My `.env` file:
```
vars {
  test1: content1
  test2: content2
}
```

**Comment by mjhcorporate** [comment]
I confirmed that this fix is working in the newest version. I did, however, find a different issue. @mikestok , could you confirm as well and close this issue? I will create a new issue for my finding.

**Comment by mjhcorporate** [comment]
New finding: https://github.com/usebruno/bruno/issues/1724

**Comment by mikestok** [comment]
I see this is fixed in v1.10.0. Thank you.
